infrastructure/images/moduletest-reset-jobs/Taskfile.yml

```diff
@@ -0,0 +1,26 @@
+# Lagoon Redeploy Controller Taskfile (task.dev)
+#
+# Builds, tests and pushes builds of container-images for the Lagoon Redeploy Controller
+
+version: '3'
+
+vars:
+  repository: '{{.IMAGE_URL | default "ghcr.io/danskernesdigitalebibliotek/dpl-platform/moduletest-file-reset-job"}}'
+  tag: '{{.IMAGE_TAG | default "latest"}}'
+  image: "{{.repository}}:{{.tag}}"
+
+tasks:
+  build:
+    desc: Build an image for the Moduletest File Reset Job
+    summary: Tag can  be specified via the tag environment variable. If omitted the build will assume "latest".
+    cmds:
+      - docker build -t {{.image}} .
+
+  push:
+    desc: Push a build of the shell to {{.repository}}
+    vars:
+      latest: "{{.repository}}:latest"
+    cmds:
+      - docker tag {{.image}} {{.latest}}
+      - docker push {{.image}}
+      - docker push {{.latest}}
```

Comment:
I don't think you need this, as we never really wanna push around the CI, do we?

Reply:
thank you